src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $animation-time;
```

Comment:
yes in our case it is only two properties and it is shorter, but for the future it is better to name exact properties that will change on transition, to avoid mistakes, when the behaviour of our code is unexpectred

Reply:
Okay, thank you! I will know in the future)